templates/slim/helpers.rb

```diff
@@ -3,14 +3,13 @@
 require 'asciidoctor'
 require 'json'
 
+# Needed only in compile-time.
+require 'slim-htag' if defined? Slim
```

Comment:
This is not required, I’ve added it to my `helpers.rb` just to allow using Slim templates even directly without ahead of time compilation.

Reply:
For development it might be useful to avoid the build step. Worth keeping as currently scoped in the `unless opal` block. I'm very open to remove it if there's a good argument against though.

Reply:
As you said, it might be useful in development, for someone.